app/src/main/java/org/oppia/app/application/ApplicationComponent.kt

```diff
@@ -36,7 +37,7 @@ import org.oppia.util.threading.DispatcherModule
     ContinueModule::class, FractionInputModule::class,
     ItemSelectionInputModule::class, MultipleChoiceInputModule::class,
     NumberWithUnitsRuleModule::class, NumericInputRuleModule::class,
-    TextInputRuleModule::class, DragDropSortInputModule::class,
+    TextInputRuleModule::class, DragDropSortInputModule::class, ImageClickInputModule::class,
```

Comment:
Can you move this to the next line to maintain consistency in the way the modules are listed?

Reply:
will do

Reply:
done